### PR TITLE
Mandatory de-activation of forced segwit deployment

### PR DESCRIPTION
**Pull request by joshuayabut**
This BIP supercedes BIP148 and outlines the methods and actions necessary to prevent unwanted network segmentation and forced isolation caused by non-consensual BIP148 and Segregated Witness deployment.

### CODE REVIEW

**Comment by achow101** [comment]
First of all, the other commits which are unrelated to your BIP should be removed and not part of this pull request. They do not belong here.

Secondly, this BIP number does not seem to have been assigned by the BIP editor. You are not allowed to assign your own BIP number; they will be assigned by the BIP editor if it meets the BIP standard.

**Comment by luke-jr** [comment]
This seems to just be trolling. In any case, the BIP process requires public discussion on the bitcoin-dev mailing list prior to submission here. It also requires a reasonable copyright licence, so BIPs can be shared without restrictions. If this is to become a serious proposal, the trolling should be removed, a proper license chosen, and it must be discussed on the mailing list first.

Additionally, as @achow101 pointed out, there is no BIP number assigned for this, nor will there be until the minimum criteria have been met.

See BIP 2 for details on the BIP process.

**Comment by joshuayabut** [comment]
Ok I will implement this without your approval.

**Comment by kek-coin** [comment]
So you wanted to sell [a 5 year old "0day"](https://bitcointalk.org/index.php?topic=114751.0) for 500 BTC? Well, can't blame you for trying.

File: bip-0200.mediawiki [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The Bitcoin protocol was initially designed as a decentralized standard. Unfortunately BIP148 establishes and assigns significant standards-implementation authority to large and more influential commercial organizations and away from independent parties. The author of this proposal fully rejects BIP148 as being contrary to the spirit of Bitcoin and rejects the premise that overall Bitcoin consensus can be gained through a simple voting mechanism.
+
+BIP148 (UASF) additionally creates the scaffolding required for large influential groups to take ownership of the global Bitcoin protocol which is a significant departure from the original decentralized nature of Bitcoin. The presence of this scaffolding creates extraordinary risk of centralization within the Bitcoin community and must be resisted.
+
+This BIP takes a red-team-centric approach and proposes multiple legally acceptable covert and overt means to resist and eradicate BIP148 and SegWit from the global Bitcoin network in the event it is non-consensually activated.
```
Comment (mecampbellsoup) [inline bip-0200.mediawiki]: I think you are completely mischaracterizing the SegWit debate here. 

How do you expect the network to reach consensus when the majority of hashing power opposes SegWit SF activation due to the fact that it removes their ability to utilize covert ASICBoost? Why do you think, after all this time, SegWit has still not been activated by miners (where "miners" means essentially the 2 or 3 largest in the business which control the majority of hashing power)? 

These miners have "agreed" to the so-called Barry Silbert Accords from Consensus 2017. But I would be very surprised if push came to shove and those miners actually activated SegWit SF as [that plan](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-March/013921.html) calls for. They have been stalling for nearly 2 years now, dragging the core developers along with them. 

There is no reason to be opposed to SegWit at this point as it's already been activated on Litecoin and the technical FUD has been debunked. SegWit effectively gives us bigger blocks (i.e. more transactions per block), something the miners point to as the reason they have been clamoring for an increased 2MB blocksize limit. 

I'm amazed that @luke-jr and @achow101 took the time out of their respective days to even respond to this piece of sh*t trolling effort.

**Comment by dabura667** [comment]
> Ok I will implement this without your approval instead of jumping through your hoops Mr. UASF man.

You knew how the BIP process worked before making this.

You are just trolling so you can link this issue on /r/btc and get that sweet sweet karma.

I hope you get all the karma you wanted. Looks like you put a ton of effort into this troll.

Why don't you try submitting as a BUIP to Bitcoin Unlimited?

**Comment by luke-jr** [comment]
Please don't feed the troll.

**Pull request closed without merging**